JenkinsfileStressTests

```diff
@@ -0,0 +1,83 @@
+def maxRetries = 120
+def waitForCommandSuccess(command, maxRetries) {
+    def tries = 0;
+    while (tries++ < maxRetries) {
+        try {
+            sh command
+            return true
+        } catch (Exception e) {
+            if (tries >= maxRetries) {
+                throw new Exception("${maxRetries} tries exceeded. ${command} failed.", e)
+            }
+            sh "sleep 1"
+        }
+    }
+}
+
+pipeline {
+    agent { label 'my-slave' }
+
+    tools {
+        maven 'maven'
+    }
+
+    stages {
+        stage('Compile') {
+            steps {
+                sh "mvn clean install -DskipTests"
+            }
+        }
+        stage('Build image') {
+            steps {
+                script {
+                    docker.withTool('docker') {
```

Comment:
It's ok for now but we really need to use jib to avoid repeating that kind of code

Reply:
I agree with you, but it's another issue.